it/managed-tests/invoker.properties

```diff
@@ -2,8 +2,12 @@ invoker.goals.1 = dependency:tree -Pmanaged-testng -Pol
 invoker.goals.2 = clean install -Pmanaged-testng -Pol
 invoker.goals.3 = dependency:tree -Pmanaged-junit -Pol
 invoker.goals.4 = clean install -Pmanaged-junit -Pol
+invoker.goals.5 = dependency:tree -Pmanaged-junit5 -Pol
+invoker.goals.6 = clean install -Pmanaged-junit5 -Pol
 
-invoker.goals.5 = dependency:tree -Pmanaged-testng -Pwlp
-invoker.goals.6 = clean install -Pmanaged-testng -Pwlp
-invoker.goals.7 = dependency:tree -Pmanaged-junit -Pwlp
-invoker.goals.8 = clean install -Pmanaged-junit -Pwlp
\ No newline at end of file
+invoker.goals.7 = dependency:tree -Pmanaged-testng -Pwlp
+invoker.goals.8 = clean install -Pmanaged-testng -Pwlp
+invoker.goals.9 = dependency:tree -Pmanaged-junit -Pwlp
+invoker.goals.10 = clean install -Pmanaged-junit -Pwlp
+invoker.goals.11 = clean install -Pmanaged-junit5 -Pwlp
+invoker.goals.12 = clean install -Pmanaged-junit5 -Pwlp
```

Comment:
I think this one is supposed to be `dependency:tree` instead of `clean install`.